cves/kernel/CVE-2022-1975.yml

```diff
@@ -55,7 +55,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The vulnerability occurred when the Linux kernel didn't prevent
+  context switches during certain context operations that should have been
+  resolved uninterrupted. An incorrect flag was set that gave instructions on
+  how memory allocation should be performed. It allowed sleeping during the
+  memory allocation which could leave the process suspended. This made it
+  possible for the context to switch during a vulnerable state which
+  triggers an expectation that goes uncaught.
```

Comment:
Could be less technical and explained more. For example what content switches and context operations are.

Reply:
It has been simplified and explained better (Hopefully).